test/blackbox-tests/test-cases/melange/emit-with-runtime-deps-edge-cases.t

```diff
@@ -137,17 +142,87 @@ Test depending on external paths
   >  (alias mel)
   >  (target external-output)
   >  (emit_stdlib false)
+  >  (libraries melange.node)
   >  (preprocess (pps melange.ppx))
   >  (runtime_deps /etc/hosts))
   > EOF
   $ cat > external/main.ml <<EOF
-  > let dirname = [%bs.raw "__dirname"]
+  > let dirname = [%mel.raw "__dirname"]
   > let file_path = "./assets/file.txt"
   > let file_content = Node.Fs.readFileSync (dirname ^ "/" ^ file_path) \`utf8
   > let () = Js.log file_content
   > EOF
 
   $ dune build @mel --display=short
+          melc external/external-output/node_modules/melange.js/caml.js
```

Comment:
since you mentioned it, is this list useful or should we pipe that into `grep -v node_modules`?

Reply:
I think we can remove `--display` flag altogether, if @anmonteiro is ok with it.